src/store/playlist-store.js

```diff
@@ -1,12 +1,36 @@
-import { makeAutoObservable } from "mobx";
+import { makeAutoObservable, flow } from "mobx";
 
 class PlaylistStore {
-  myPlaylist = [];
-
   constructor(rootStore) {
     this.rootStore = rootStore;
 
-    makeAutoObservable(this);
+    this.myPlaylists = {};
+
+    makeAutoObservable(this, {
+      fetchMyPlaylists: flow,
+    });
+  }
+
+  setMyPlaylists(response) {
+    this.myPlaylists = response;
+  }
+
```

Comment:
"fech()", a function in JS for interacting with the database, especially with API's. ie, fetching dating, uploading data, editing data, and deleting data.